auto_sizing/size_calculation.py

```diff
@@ -117,7 +115,7 @@ def calculate_sample_sizes(
 
         return result_dict
 
-    def publish_results(self, result_dict: Dict[str, float], current_date: str) -> None:
+    def publish_results(self, result_dict: Dict[str, Any], current_date: str) -> None:
```

Comment:
Agree, I was trying stuff like that, but I think this was one that caused issues in other downstream functions, that required this same disgusting type hint to be passed there. I, for one, vote for allowing Any rather than this abomination, lol